test/test_helper.rb

```diff
@@ -149,4 +149,24 @@ def create_and_login_additional_users(num)
       click_button 'Login'
     end
   end
+
+  def admin_order_setup(status)
+    admin = User.create(username: 'admin',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.